src/Rector/Deprecation/Base/EntityViewBase.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace DrupalRector\Rector\Deprecation\Base;
+
+use DrupalRector\Utility\AddCommentTrait;
+use PhpParser\Node;
+use Rector\Core\Rector\AbstractRector;
+
+/**
+ * Replaced deprecated entity_load() calls.
+ *
+ * See https://www.drupal.org/node/2266845 for change record.
+ *
+ * What is covered:
+ * - Static replacement
+ * - The reset parameter is handled with a ternary operator
+ *
+ * Improvement opportunities
+ * - Dependency injection
+ */
+abstract class EntityViewBase extends AbstractRector
+{
+
+  use AddCommentTrait;
+
+  /**
+   * @inheritdoc
+   */
+  public function getNodeTypes(): array
+  {
+    return [
+      Node\Expr\FuncCall::class,
+    ];
+  }
+
+  /**
+   * @inheritdoc
+   */
+  public function refactor(Node $node): ?Node
+  {
+
+    if ($this->getName($node->name) !== 'entity_view') {
+      return NULL;
+    }
+
+    $name = new Node\Name\FullyQualified('Drupal');
+
+    $entity_type_method_name = new Node\Identifier('entityTypeManager');
+
+    $var = new Node\Expr\StaticCall($name, $entity_type_method_name);
+
+    $getViewBuilder_method_name = new Node\Identifier('getViewBuilder');
+
+    $entityRef = $node->args[0]->value;
```

Comment:
Could we make this `$entity_reference`? I like how the names you've used are mostly matching the method names like `$getViewBuilder_method_name` which is very clear to me. :)